host_applications/linux/apps/gencmd/gencmd.c

```diff
@@ -36,6 +36,20 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "interface/vmcs_host/vc_vchi_gencmd.h"
 
+void show_usage()
+{
+   printf( "Usage: vcgencmd [-t] command\n" );
```

Comment:
@pelwell / @popcornmix What's the `instNum` for? Should that be documented in the usage string too?

Reply:
Absolutely nothing at all - it's a function-local variable which is assigned to and never used.

Reply:
Cool, that's what I thought, but as this is code which talks to firmware I didn't know if there was some deeper magic going on :wink: